mhs/outbound/outbound/request/synchronous/handler.py

```diff
@@ -26,6 +31,93 @@ def initialize(self, workflows: Dict[str, workflow.CommonWorkflow],
 
     @timing.time_request
     async def post(self):
+        """
+        ---
+        summary: Make a request to the MHS
+        description: Make a request to the MHS
+        operationId: postMHS
+        parameters:
+          - name: Interaction-Id
+            in: header
+            required: true
+            schema:
+              type: string
+            description: ID of the interaction that you want to invoke. e.g. QUPC_IN160101UK05
+          - name: sync-async
+            in: header
+            required: true
+            schema:
+              type: string
+              enum: ["true", "false"]
```

Comment:
So this looks a bit odd, but when I took a look through the [OpenAPI](https://swagger.io/specification) spec, I couldn't figure out how a boolean would get serialised for certain. So I've done this like this. I'm not against changing it back, if we know this for certain.